src/modules/account/views/ViewAccountDetails.vue

```diff
@@ -96,70 +80,81 @@
     </div>
   </div>
 </div>
-
 </template>
 
 <script>
-import { ref, inject } from 'vue';
-import { useRouter } from "vue-router";
-import TextInput from '@/components/TextInput.vue';
-import PasswordInput from '@/components/PasswordInput.vue';
-import { copyKeyFunc } from '@/util/functions';
-import { useToast } from "primevue/usetoast";
+import {ref, computed} from "vue";
+import {useRouter} from "vue-router";
+import TextInput from "@/components/TextInput.vue";
+import PasswordInput from "@/components/PasswordInput.vue";
+import {copyToClipboard } from '@/util/functions';
+import {useToast} from "primevue/usetoast";
+import {walletState} from "@/state/walletState";
+import {Helper} from "@/util/typeHelper";
+import {networkState} from "@/state/networkState";
+import {WalletUtils} from "@/util/walletUtils";
 
 export default {
-  name: 'ViewAccountDetails',
+  name: "ViewAccountDetails",
   components: {
-    TextInput, PasswordInput
+    TextInput,
+    PasswordInput
   },
   props: {
-    address: String,
+    address: String
   },
 
-  setup(p){
+  setup(p) {
     const toast = useToast();
-    const appStore = inject("appStore");
     const router = useRouter();
 
     // get account details
-    const acc = appStore.getAccDetailsByAddress(p.address);
-    console.log(acc)
-    if(acc === -1){
-      router.push({ name: "ViewAccountDisplayAll"});
+    const acc = walletState.currentLoggedInWallet.accounts.find((add) => add.address == p.address);
+    if (acc === -1) {
+      router.push({name: "ViewAccountDisplayAll"});
     }
-    acc.pretty = appStore.pretty(acc.address);
+    acc.pretty = Helper.createAddress(acc.address).pretty();
```

Comment:
you are adding a new property pretty in the walletAccount instance, maybe use a temporary variable to store the value